back-end/routes/chain/get_account.js

```diff
@@ -17,41 +14,9 @@ module.exports = async function(req, res) {
         res.send({ message: `Not found account with account name ${accountName}` });
     }
 
-    const query = `account:eosio action:neworg (data.active.accounts.permission.actor:${accountName} OR data.owner.accounts.permission.actor:${accountName})`;
-    let transactionRes = await eosio.dfuseClient.searchTransactions(query);
-    // get array of account names from dfuse tranactions log
-
-    let orgInfo = [];
-    if (transactionRes.transactions) {
-        const accountNamesInfo = transactionRes.transactions
-            .filter(transaction => (transaction.lifecycle.transaction_status === "executed" && transaction.lifecycle.pub_keys))
-            .map(transaction => {
-                const transactionItem = transaction.lifecycle;
-                const accountName = transactionItem.execution_trace.action_traces[0].act.data.name;
-                return accountName;
-            });
-
-        if (!accountNamesInfo || accountNamesInfo.length === 0) {
-            res.status(404);
-            res.send({ message: `Not found account from dfuse with account name ${accountName}` });
-            return; // not sure if this is needed...
-        }
-        // get name from account name using mongodb
-        orgInfo = await Promise.all(accountNamesInfo.map(async accountName => {
-            const orgDoc = await accountController.findOne({ accountName: accountName });
-            if (!orgDoc) {
-                res.status(404);
-                res.send({ message: `Not found account with account name ${accountName}` });
-                return; // not sure if this is needed...
-            }
-            return { name: orgDoc.name, accountName: accountName }
-        }))
-    }
-
     const accountDocInfo = {
         accountType: accountDoc.accountType,
-        name: accountDoc.name,
-        organizations: orgInfo
+        name: accountDoc.name
```

Comment:
can we use commonName and accountName in database and variables to avoided confusion of which this is

Reply:
```
    const accountDocInfo = {
        accountType: accountDoc.accountType,
        accountName: accountDoc.name
    };

```

is that what you mean?